vignettes/cgdsMigration.Rmd

```diff
@@ -17,149 +18,295 @@ output:
 knitr::opts_chunk$set(cache = TRUE)
 ```
 
+<!-- styling  with nowrap DT row -->
+<style>
+table {
+  white-space: nowrap;
+}
+</style>
+
 # Introduction
 
-This vignette aims to help developers migrate from the now defunct `cgds`
-CRAN package. Note that the `cgds` package code is shown for comparison but it
+This vignette aims to help developers migrate from the now defunct `cgdsr`
+CRAN package. Note that the `cgdsr` package code is shown for comparison but it
 is not guaranteed to work. If you have questions regarding the contents,
-please create an issue at the GitHub repository:
-https://github.com/waldronlab/cBioPortalData/issues
+please create an issue at the [GitHub](https://github.com/waldronlab/cBioPortalData/issues) repository.
+
+
+# Loading the packages
+
+```{r, warning=FALSE, comment=FALSE, message=FALSE}
+## Import needed libraries
+if (!require("devtools")) install.packages("devtools")
+if (!require("BiocManager")) install.packages("BiocManager")
+if (!require("cgdsr")) devtools::install_version("cgdsr",version="1.3")
+#if (!require("cgdsr")) devtools::install_github("kmezhoud/cgdsr")
+if (!require("cBioPortalData")) BiocManager::install("cBioPortalData")
+#if (!require("cBioPortalData")) devtools::install_version("cBioPortalData",version="2.7.11")
+if (!require("dplyr")) install.packages("dplyr")
+if (!require("tidyr")) install.packages("tidyr")
+if (!require("DT")) install.packages("DT")
+if (!require("tictoc")) install.packages("tictoc")
 
-# Loading the package
 
-```{r,include=TRUE,results="hide",message=FALSE,warning=FALSE}
-library(cBioPortalData)
 ```
 
-# Discovering studies
+# Discovering studies {.tabset .tabset-fade .tabset-pills}
 
-## cgds
+## `cgdsr`
+```{r, comment=FALSE }
+cgds <- cgdsr::CGDS("http://www.cbioportal.org/")
+
+getCancerStudies.CGDS(cgds) %>%
+    datatable(options = list(pageLength = 5))
```

Comment:
Do not evaluate `cgdsr` code chunks. This package is defunct and not available on CRAN. The code provided is for demo purposes only.